### PR TITLE
Improve gfortran ROOT detection

### PR DESCRIPTION
**Pull request by dberzano**
Fix inconsistencies introduced with #1563

### CODE REVIEW

**Comment by dberzano** [comment]
@mpuccio fyi

**Comment by dberzano** [comment]
This fixes a problem building ROOT 5 reported on the mailing lists.

File: root.sh [hunk 1]
```diff
@@ -37,13 +37,12 @@ incremental_recipe: |
   JOBS=$((${JOBS:-1}*3/5))
   [[ $JOBS -gt 0 ]] || JOBS=1
 
-  cmake --build . -- ${JOBS:+-j$JOBS} install
-  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+  cmake --build . --target install ${JOBS:+-- -j$JOBS}
+  rm -vf "$INSTALLROOT/etc/plugins/TGrid/P010_TAlien.C"         \
```
Comment (aphecetche) [inline root.sh]: How is this related to gfortran ?

Comment (dberzano) [reply root.sh]: It's not, and it's not changing the behaviour (those are syntax improvements), I won't open a second PR just for that 😄
